src/main/java/org/broadinstitute/hellbender/tools/exome/hmm/CopyNumberTriStateHiddenMarkovModelArgumentCollection.java

```diff
@@ -0,0 +1,93 @@
+package org.broadinstitute.hellbender.tools.exome.hmm;
+
+import org.apache.commons.lang.math.DoubleRange;
+import org.broadinstitute.hellbender.cmdline.Argument;
+import org.broadinstitute.hellbender.exceptions.UserException;
+
+/**
+ * User arguments to compose a custom {@link CopyNumberTriStateHiddenMarkovModel}.
+ *
+ * <p>
+ *     Default values are those that were show to work well with human data in paper
+ *     <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3484655/">Fromer et al. 2012</a>.
+ * </p>
+ *
+ * @author Valentin Ruano-Rubio &lt;valentin@broadinstitute.org&gt;
+ */
+public final class CopyNumberTriStateHiddenMarkovModelArgumentCollection {
+
+    public static final String EVENT_RATE_SHORT_NAME = "eventRate";
+    public static final String EVENT_RATE_FULL_NAME = "eventSegmentOpenRate";
+    public static final String MEAN_TARGETS_PER_SEGMENT_SHORT_NAME = "eventTargetCount";
+    public static final String MEAN_TARGETS_PER_SEGMENT_FULL_NAME = "meanNumberOfTargetsPerSegment";
+    public static final String MEAN_DISTANCE_BETWEEN_TARGETS_IN_SEGMENT_FULL_NAME = "meanDistanceBetweenSegmentTargets";
+    public static final String MEAN_DISTANCE_BETWEEN_TARGETS_IN_SEGMENT_SHORT_NAME = "eventTargetDistance";
+    public static final String MEAN_DELETION_COVERAGE_SHIFT_FULL_NAME = "meanDeletionCoverageSigmaShift";
+    public static final String MEAN_DELETION_COVERAGE_SHIFT_SHORT_NAME = "deletionShift";
+    public static final String MEAN_DUPLICATION_COVERAGE_SHIFT_FULL_NAME = "meanDuplicationCoverageSigmaShift";
+    public static final String MEAN_DUPLICATION_COVERAGE_SHIFT_SHORT_NAME = "duplicationShift";
+
+    @Argument(doc = "Probability that a target is the first in a deletion or duplication segment",
+              fullName = EVENT_RATE_FULL_NAME,
+              shortName = EVENT_RATE_SHORT_NAME,
+              optional = true)
+    public double eventSegmentRate = 1.0e-10;
```

Comment:
Sure, bad default, will change. 

Yet even 1e-8 is quite small if you think about it.  If you have around ~100,000 targets per sample that means that you expect to see only 1 event every 100 samples. So it makes little sense and I guess is its compensating for false positives that happen in practice; perhaps some kind of under-the-counter multiple-test correction?

Also I think is wrong to try to use targets as a unit here as different capture or panels require to recalculate this value. I think we should eventually move into using base-pairs rather than targets which we can maintain constant per species.